lib/picoruby/build.rb

```diff
@@ -40,6 +40,18 @@ def picoruby(alloc_libc: true)
         cc.defines << define if cc.defines.none? { _1.start_with? key }
       end
 
+      timestamp = Time.at(`git log -1 --format=%ct`.to_i).utc.strftime('%Y-%m-%dT%H:%M:%SZ')
+      branch = `git branch --show-current`.strip
+      commit_hash = `git log -1 --format=%h`.strip
+
+      File.write(
+        "./src/version.c",
+        File.read("./src/version.c.in")
```

Comment:
I want you to use absolute paths instead of relative paths to avoid unexpected behavior

Reply:
I am sorry that I am not aware of the existence of the constant MRUBY_ROOT. I used it and fixed it so that it is an absolute path.